lib/rss.js

```diff
@@ -0,0 +1,203 @@
+const RSS = require('rss')
+const generateDiff = require('deep-diff').diff
+const fs = require('fs')
+
+function createLeadersHash(array) {
+  return array.reduce((hash, elem) => {
+    hash[elem.id] = elem
+    return hash
+  }, {})
+}
+
+function createOrgHash(array) {
+  let rank = 0
+  return array.reduce((hash, elem) => {
+    rank++
+    elem.rank = rank
+    elem.leaders = createLeadersHash(elem.leaders)
+    hash[elem.slug] = elem
+    return hash
+  }, {})
+}
+
+let current = {}
+try {
+  current = JSON.parse(
+    fs.readFileSync(`${__dirname}/../out/feed_items.json`).toString()
+  )
+} catch (e) {
+  current = {}
+}
+
+const buildInfo = JSON.parse(
+  fs.readFileSync(`${__dirname}/../out/scrape_info.json`).toString()
```

Comment:
So we don't need to fetch this from the network?

You can use the modified date from the json file, I think we do that for the "Last updated" information.

Reply:
Ok, so instead of having a `scrape_info.json` file we should check the modified time on the `data.json` file using `fs.stat()`?

Reply:
Yes, for now.

We'll modularize it once this is done.